content/en/docs/content-structure/layout-builder/_index.md

```diff
@@ -31,8 +31,22 @@ Expandable pairs of question/answer or header/section fields.
 
 | Label | Machine Name | Type                                 | Required | Cardinality | Help text | Field Settings | Notes |
 |-------|--------------|--------------------------------------|----------|-------------|-----------|----------------|-------|
-| Title | field_title  | Text (plain)                         | yes      | 1           |
-| Body  | field_body   | Text (formatted, long, with summary) | no       | 1           |
+| Heading (H4) | field_accordion_heading  | Text (plain)                         | yes      | 1           |
+| Body  | field_accordion_body   | Text (formatted, long, with summary) | no       | 1           |
```

Comment:
I think this machine name should be consistent across components, like `field_body`